plugins/pulp_ostree/plugins/distributors/configuration.py

```diff
@@ -76,4 +76,10 @@ def get_repo_relative_path(repo, config):
     :return: relative path for the repository
     :rtype:  str
     """
-    return repo.id
+    path = config.get(constants.DISTRIBUTOR_CONFIG_KEY_RELATIVE_PATH)
+    if path:
+        if path.startswith('/'):
+            path = path[1:]
+    else:
+        path = repo.id
+    return path
```

Comment:
Oh, it does look like you did this server-side.

I do still think there's a problem with also doing it client-side, namely during times where the user updates the feed, but does not also supply the same relative URL. I think that will lead to results that surprise users.